vss-extension.json

```diff
@@ -117,42 +117,76 @@
                     "displayName": "Marketplace URL",
                     "value": "https://marketplace.visualstudio.com"
                 },
-                "authenticationSchemes": [
+              "authenticationSchemes": [
+                {
+                  "type": "ms.vss-endpoint.endpoint-auth-scheme-token",
+                  "inputDescriptors": [
                     {
-                        "type": "ms.vss-endpoint.endpoint-auth-scheme-basic",
-                        "inputDescriptors": [
-                            {
-                                "id": "username",
-                                "name": "Username",
-                                "description": "Username",
-                                "inputMode": "textbox",
-                                "isConfidential": false,
-                                "validation": {
-                                    "isRequired": false,
-                                    "dataType": "string",
-                                    "maxLength": 300
-                                },
-                                "values": {
-                                    "inputId": "username",
-                                    "isDisabled": true,
-                                    "defaultValue": ""
-                                }
-                            },
-                            {
-                                "id": "password",
-                                "name": "Personal access token",
-                                "description": "Team Services personal access token. Requires at least the Marketplace (publish) scope.",
-                                "inputMode": "passwordbox",
-                                "isConfidential": true,
-                                "validation": {
-                                    "isRequired": true,
-                                    "dataType": "string",
-                                    "maxLength": 300
-                                }
-                            }
-                        ]
+                      "id": "apitoken",
+                      "name": "Personal access token",
+                      "description": "Team Services personal access token. Requires at least the Marketplace (publish) scope.",
```

Comment:
Probably we can take advantage of this `"description"` to inform users that we are hitting the public Marketplace and if they want to hit the local gallery (onprem), that they should use the "Basic Auth" option instead. We can do the same in the `serverUri´ field (basic auth) to inform that Basic Auth it should be used only for TFS On Prem, and they should use the "Token" option if they want to hit the public Gallery. I can tweak the descriptions before merge if you prefer.